README.md

```diff
@@ -625,6 +625,13 @@ If you want the docker container to run in the background pass the `-d` flag rig
 You can adjust the maximum cache size by appending `--max_size N`, where N is
 the maximum size in Gibibytes.
 
+### Docker Compose notes
+
+* See [examples/docker-compose.yml](examples/docker-compose.yml) for an example configuration.
+
+* To overwrite the default maximum cache size of the prebuilt image use the `command` or
+  build your own image as described below
```

Comment:
Thanks for the contribution.

The second point is a little confusing to me, how does this sound to you:
```suggestion
See [examples/docker-compose.yml](examples/docker-compose.yml) for an example configuration (modify the `--max_size` flag in there to suit your needs).
```